Specs/NumberWithUnit/Portuguese/CurrencyModel.json

```diff
@@ -1536,13 +1536,13 @@
   },
   {
     "Input": "custou apenas 15 dólares e 15 centavos.",
-    "NotSupported": "javascript, python, java",
+    "NotSupported": "javascript, java",
     "Results": [
       {
         "Text": "15 dólares e 15 centavos",
         "TypeName": "currency",
         "Resolution": {
-          "value": "15,15",
+          "value": "15.15",
```

Comment:
You should not modify the separators. Please revert these changes so we can merge.

Reply:
This is a legacy design decision, that hopefully will be finally addressed and uniformly sorted out in release 2.0. But that will take a while.

Reply:
No problem, reverted those changes

Reply:
There is only one revert pending for "3,99" in the output of "3 euros com 99 centavos".

Reply:
Ah yes sorry! fixed now